scripts/ci/check_pull_request.py

```diff
@@ -79,8 +79,11 @@ def main():
 
 
 def check_pull_request(title, body):
-    if title.startswith('['):
-        error_flag = regex_line(title)
+    if title.startswith('[') or title.startswith('{'):
+        if title.startswith('['):
+            error_flag = regex_line(title)
+        else:
+            error_flag = False
```

Comment:
How about we make it like this:

```suggestion
        error_flag = False
        if title.startswith('['):
            error_flag = regex_line(title)            
```

This means we first initialize `error_flag` as `False`, and we change it to `True` when error occurs. This increases readability. 😊